README.md

```diff
@@ -62,6 +62,8 @@ This is the home of [ar.io] SDK. This SDK provides functionality for interacting
   - [ANT APIs](#ant-apis)
     - [`init({ processId, signer })`](#init-processid-signer-)
     - [`getInfo()`](#getinfo-1)
+  - [`getHandlers()`](#gethandlers)
+  - [`validate({ validations })`](#validate-validations-)
```

Comment:
these are in the correct spot - please fix headers

Reply:
https://github.com/ar-io/ar-io-sdk/pull/237/commits/395f7fb01927cc0a247b521d0f992eef00278348